docs/content/modeling/public-access.mdx

```diff
@@ -115,18 +116,52 @@ Let us create a relationship tuple that states: **any user can view document:com
   ]}
 />
 
-### 02. Check That The Relationship Exists
+:::caution Wildcard syntax usage
 
-Once the above _relationship tuple_ is added, we can <ProductConcept section="what-is-a-check-request" linkName="check" /> if **bob** cab `view` `document`:**company-psa.doc**. <ProductName format={ProductNameFormat.ShortForm}/> will return `{ "allowed": true }` even though no relationship tuple linking **bob** to the document was added. That is because the relationship tuple with `*` as the user made it so everyone can `view` the document, making it public.
+Please note that type-bound public access is not a wildcard or a regex expression.
 
-<CheckRequestViewer user={'user:bob'} relation={'view'} object={'document:company-psa.doc'} allowed={true} />
+**You cannot use the `<type>:*` syntax in the tuple's object field.**
+
+The following syntax is invalid:
+
+<RelationshipTuplesViewer
+  relationshipTuples={[
+    {
+      _description: 'It is invalid to use this syntax in the object field. The below relationship tuple is invalid and does not mean that Bob can view all documents.',
+      user: 'user:bob',
+      relation: 'view',
+      object: 'document:*',
+    },
+  ]}
+/>
+
+:::
 
 :::caution Wildcard syntax usage
-Please note that `*` is a special <ProductName format={ProductNameFormat.ShortForm}/> syntax meaning **everyone** when used as a _user_ within a _relationship tuple_. It is not a wildcard or regex expression.
 
-**You cannot use it with a type to mean all objects in that type.** `workspace:*` does not mean all types; it means a single object with the type `workspace` and the `object_id` the string `*`.
+**You cannot use `<type>:*` as part of a userset in the tuple's user field.**
+
+The following syntax is invalid:
+
+<RelationshipTuplesViewer
+  relationshipTuples={[
+    {
+      _description: 'It is invalid to use this syntax as part of a userset. The below relationship tuple is invalid and does not mean that members of any org can view the company-psa document.',
+      user: 'org:*#member',
+      relation: 'view',
+      object: 'document:company-psa.doc',
+    },
+  ]}
+/>
+
 :::
 
+### 02. Check That The Relationship Exists
+
+Once the above _relationship tuple_ is added, we can <ProductConcept section="what-is-a-check-request" linkName="check" /> if **bob** cab `view` `document`:**company-psa.doc**. <ProductName format={ProductNameFormat.ShortForm}/> will return `{ "allowed": true }` even though no relationship tuple linking **bob** to the document was added. That is because the relationship tuple with `*` as the user made it so everyone can `view` the document, making it public.
```

Comment:
```suggestion
Once the above _relationship tuple_ is added, we can <ProductConcept section="what-is-a-check-request" linkName="check" /> if **bob** cab `view` `document`:**company-psa.doc**. <ProductName format={ProductNameFormat.ShortForm}/> will return `{ "allowed": true }` even though no relationship tuple linking **bob** to the document was added. That is because the relationship tuple with `user:*` as the user made it so every object of type user (such as `user:bob`) can `view` the document, making it public.
```